docs/modelserving/v1beta1/llm/huggingface/multi-node/README.md

```diff
@@ -129,8 +136,92 @@ spec:
       pipelineParallelSize: 2
       tensorParallelSize: 1
     }  
+    
 ```
 
+!!! info "Mechanism: How to Set GPU Allocation per Node"
```

Comment:
This entire block isn't rendering correctly on Github at least, not sure if it's a syntax bug or just a rendering issue here

Reply:
KServe website uses MkDocs, so the documentation might not render properly on GitHub. You can preview it locally by following this guide:
https://github.com/kserve/website/blob/main/docs/help/contributor/mkdocs-contributor-guide.md

I checked it with mkdocs.
![image](https://github.com/user-attachments/assets/4c97f38e-2a58-4dce-be8d-57d7b6500361)

Reply:
that looks fine